### ISSUE TITLE
update

### ISSUE THREAD

**Issue by Hardy69x** [rohKane/kane-qb-inventory#3]
Can u update with qb-shops  (No such export CreateShop in resource qb-inventory)

**Comment by rohKane** [comment]
Make sure to use the latest version of qb core and everything

**Comment by kosar-omed** [comment]
> Make sure to use the latest version of qb core and everything

You would need to update your files too buddy. The new QB-Inventory/QB-Core has changed alot of things. Please see if you can update this to make it compatible with the latest changes thanks 👍

**Comment by Tak3it1nthefac3** [comment]
> > Make sure to use the latest version of qb core and everything
> 
> You would need to update your files too buddy. The new QB-Inventory/QB-Core has changed alot of things. Please see if you can update this to make it compatible with the latest changes thanks 👍

Yeah I'm having the same issue

**Comment by rohKane** [comment]
Yeah, this script is pretty much outdated for the latest version of QBCore recipe now, Because the devs has made a lots of changes to the Inventory system.
